streams/src/test/java/org/apache/kafka/streams/state/internals/MergedSortedCacheKeyValueStoreIteratorTest.java

```diff
@@ -21,29 +21,40 @@
 import org.apache.kafka.common.utils.Bytes;
 import org.apache.kafka.streams.state.KeyValueStore;
 import org.apache.kafka.streams.state.StateSerdes;
+import org.junit.Before;
 import org.junit.Test;
 
 import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertFalse;
 
 public class MergedSortedCacheKeyValueStoreIteratorTest {
 
+    private final String namespace = "one";
+    private StateSerdes<byte[], byte[]> serdes;
+    private KeyValueStore<Bytes, byte[]> store;
+    private ThreadCache cache;
+
+    @Before
+    public void setUp() throws Exception {
+        serdes = new StateSerdes<>(namespace, Serdes.ByteArray(), Serdes.ByteArray());
```

Comment:
nit: For store and cache we do want to construct them from scratch for each test case, but I think `serdes` can also be a private final variable of the class, since it is stateless?